bool collided
string[] collision_links
trajectory_msgs/JointTrajectory free_path  
trajectory_msgs/JointTrajectory collision_path 